src/ucp/proto/proto_common.c

```diff
@@ -241,7 +237,8 @@ ucp_proto_common_get_lane_perf(const ucp_proto_common_init_params_t *params,
     ucs_assert(perf->send_post_overhead >= 0);
     ucs_assert(perf->recv_overhead >= 0);
     ucs_assert(perf->max_frag > 0);
-    ucs_assert(perf->max_frag >= hdr_size);
+    ucs_assertv(perf->max_frag >= params->hdr_size, "maz_frag=%zu hdr_size=%zu",
```

Comment:
`max_frag`

Reply:
IMO better to check "perf" here since this is what we return from this function

Reply:
sorry, i meant `maz_frag` -> `max_frag`

Reply:
fixed